controller/app/cmdline/src/cmd_line.cpp

```diff
@@ -2955,12 +3002,23 @@ int cmd_line::cmd_set_stream_format(int total_matched, std::vector<cli_argument*
 {
     std::string desc_name = args[0]->get_value_str();
     uint16_t desc_index = args[1]->get_value_int();
-    std::string new_stream_format_name = args[2]->get_value_str();
+    std::string new_stream_format = args[2]->get_value_str();
 
     uint16_t desc_type_value = avdecc_lib::utility::aem_desc_name_to_value(desc_name.c_str());
-    std::string stream_format_substring = new_stream_format_name.substr(20);
-    uint64_t stream_format_value = avdecc_lib::utility::ieee1722_format_name_to_value(("IEC..." + stream_format_substring).c_str());
+    unsigned int stream_format_index = 0;
+    uint64_t stream_format_value = 0;
     std::string stream_format;
+    
+    if(new_stream_format.at(0) == 'I')
+    {
+        std::string stream_format_substring = new_stream_format.substr(20);
+        stream_format_value = avdecc_lib::utility::ieee1722_format_name_to_value(("IEC..." + stream_format_substring).c_str());
+    }
+    else
+    {
+        stream_format_index = atoi(new_stream_format.c_str());
+        stream_format_value = avdecc_lib::utility::get_ieee1722_format_by_index(stream_format_index);
```

Comment:
Can we change this API to match the one above. ie ieee1722_format_index_to_value()?